code/src/TODO

```diff
@@ -0,0 +1,39 @@
+Each album:
+    - album cover image ✅
+    - album name ✅
+    - the name of each artist involved with a comma between ✅
+    fix the widht of the artist!✅
+
+Hover effects:
```

Comment:
is this supposed to be here? Since it's a guide for your work process, maybe it's not necessary to keep it here when the project is done?